### ISSUE TITLE
[Feature]: Separate delete messages permission

### ISSUE THREAD

**Issue by justanediblefriend** [simplex-chat/simplex-chat#4943]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### App version

5.8.1

### Feature

There is currently no way to make it so that nobody is able to delete anyone else's messages, which is an extremely crucial for many, even most, groups to have. It needs to be possible to add other members without being able to delete just anyone's messages.

**Comment by justanediblefriend** [comment]
Sorry can you explain what you mean by that question? Idk if there's a misunderstanding here but you can't, right? As for what the suggestion means, it just means make it so it's possible for people to not be able to delete one another's messages. Like, not able to remove messages...if that makes sense. Sorry, a little confused and not sure how to clarify.

**Comment by justanediblefriend** [comment]
> why do you need this to be possible?

In many spaces things are set up such that abuse from message deletion is a significantly higher risk than the risk of messages that need to be deleted.

**Comment by layered-design** [comment]
> In many spaces things are set up such that abuse from message deletion is a significantly higher risk than the risk of messages that need to be deleted.

100% agreed. If someone says something that a person with the power to delete _others_' messages doesn't like in a group chat, for example, that person can just delete the messages and pretend it didn't happen. Unfortunately, I've seen this happen a lot. I agree that the ability/permission to delete other people's messages should be separate from the ability to add people to the chat, and not bundled together under one role. 

As I understand it, in SimpleX, there is currently a tier system for roles, with certain permissions tied to each role and the ones above it. However, there are instances in which I would trust someone to add other people to a chat who would be interested in dicussing the same topics, but I would not also trust them with the ability to delete other people's messages, and would see this as a potential risk.

**Comment by justanediblefriend** [comment]
Yeah this is one thing that exists in Signal but Simplex hasn't caught up yet. In Signal you can make it so nobody can delete anyone's messages period, and in many spaces that's necessary.

**Comment by justanediblefriend** [comment]
Oh, thought nobody would include the owners and admins. But yeah the feature would be making it so nobody, including the owners and admins, can remove messages.
